### PR TITLE
Init admin settings general appearance section

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>acceptance oCIS</strong> https://drone.owncloud.com/owncloud/web/32556/63/1
:boom: The oCISSharingInternal2 tests pipeline failed. The build has been cancelled.

**Comment by JanAckermann** [comment]
Stupid sonarCloud.... demanding on imports are conditions that have to be covered -.-

**Comment by JammingBen** [comment]
I probably need some oCIS branch checked out to make it work...? Because uploading does not work for me.

> Stupid sonarCloud.... demanding on imports are conditions that have to be covered -.-

Yeah, that's super annoying :( Especially with the composition API I feel that SonarCloud does a terrible job at analyzing coverage.

**Comment by ownclouders** [comment]
Results for <strong>acceptance oC10</strong> https://drone.owncloud.com/owncloud/web/32606/75/1
:boom: The oC10IntegrationApp2 tests pipeline failed. The build has been cancelled.

File: config/vite_ocis/config.json [hunk 1]
```diff
@@ -16,6 +16,14 @@
     "external",
     "admin-settings"
   ],
+  "options" : {
+    "previewFileMimeTypes": [
+      "image/gif",
+      "image/png",
+      "image/jpeg",
+      "text/plain"
```
Comment (kulmann) [inline config/vite_ocis/config.json:24]: ocis supports more mime types:
```json
"image/gif",
"image/png",
"image/jpeg",
"text/plain",
"image/tiff",
"image/bmp",
"image/x-ms-bmp"
```

File: packages/web-app-admin-settings/src/components/General/AppearanceSection.vue [hunk 2]
```diff
@@ -0,0 +1,104 @@
+<template>
+  <div>
+    <h2 class="oc-py-s" v-text="$gettext('Appearance')" />
+    <div>
+      <div class="oc-flex oc-flex-middle">
+        <h3>Logo</h3>
```
Comment (kulmann) [inline packages/web-app-admin-settings/src/components/General/AppearanceSection.vue:6]: missing translation

File: config/vite_ocis/config.json (hunk 1 above)
Comment (JanAckermann) [reply config/vite_ocis/config.json:24]: Ok, I'll follow up all configs later on

File: packages/web-app-admin-settings/src/mixins/general/resetLogo.ts [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      this.$_resetLogo_reset()
+    },
+
+    async $_resetLogo_reset() {
+      try {
+        const accessToken = this.$store.getters['runtime/auth/accessToken']
+        const httpClient = clientService.httpAuthenticated(accessToken)
+        await httpClient.delete('/branding/logo')
+        this.showMessage({
+          title: this.$gettext('Logo was reset successfully')
```
Comment (kulmann) [inline packages/web-app-admin-settings/src/mixins/general/resetLogo.ts:34]:
```suggestion
          title: this.$gettext('Logo was reset successfully. Reloading page...')
```
We should announce the page reload

File: packages/web-app-admin-settings/src/views/General.vue [hunk 4]
```diff
@@ -2,8 +2,9 @@
   <div>
     <app-template ref="template" :breadcrumbs="breadcrumbs">
       <template #mainContent>
-        <div class="oc-px-m">
+        <div class="oc-px-m oc-">
```
Comment (kulmann) [inline packages/web-app-admin-settings/src/views/General.vue:5]: broken class ;-)

File: packages/web-pkg/src/components/ContextActions/ActionMenuItem.vue [hunk 5]
```diff
@@ -9,44 +9,56 @@
...
-        class="oc-files-context-action-shortcut"
-        v-text="action.shortcut"
-      />
-      <span
-        v-if="action.opensInNewWindow"
-        data-testid="action-sr-hint"
-        class="oc-invisible-sr"
-        v-text="$gettext('(Opens in new window)')"
-      />
+      <div class="oc-flex oc-flex-middle oc-width-1-1">
```
Comment (kulmann) [inline packages/web-pkg/src/components/ContextActions/ActionMenuItem.vue:12]: That's also done in https://github.com/owncloud/web/pull/8398 right? Could you avoid the conflicting change? Or do you really need the wrapping divs here?

**Review by kulmann — CHANGES_REQUESTED** [review]

**Review by kulmann — APPROVED** [review]
🥳

**Pull request merged**